tests/unittests/Foundation/NSPredicateTests.m

```diff
@@ -113,20 +112,17 @@ - (instancetype)initWithValue:(NSInteger)value {
 
     NSPredicate* unArchivedPredicate = [NSKeyedUnarchiver unarchiveObjectWithData:data];
     ASSERT_TRUE_MSG(unArchivedPredicate != nil, "FAILED: unArchivedPredicate should be non-null!");
-
-    [predicate release];
 }
 
 TEST(NSPredicate, copy) {
-    NSPredicate* predicate = [[NSPredicate alloc] init];
+    NSPredicate* predicate = [NSPredicate predicateWithFormat:@"1<3"];
```

Comment:
>1<3"] [](start = 64, length = 5)

Is `TRUE` a valid predicate? If so, it might be more straightforward to just use that.